tests/unit/scatter-test.js

```diff
@@ -63,3 +63,38 @@ test('Margins are the right size when showLegend is no', function(assert) {
   assert.equal(component.get('marginRight'), 30, 'right margin exists');
   assert.equal(component.get('marginBottom'), 0, 'no bottom margin if showLegend is no');
 });
+
+/**
+ * Sets up `scatterContent` to be used in the domaing padding test.
+**/
+function setupDomainPaddingTest() {
+  scatterContent.graphicLeft = 40;
```

Comment:
Hm, I think that's bad. All tests that are defined after these two inherit this extra state. This really shouldn't be shared between tests. Use `_.cloneDeep` or  `_.merge` or something to create a new copy/add new options to it or something